D/Doom/build_tarballs.jl

```diff
@@ -46,4 +47,4 @@ dependencies = [
 
 # Build the tarballs, and possibly a `build.jl` file.
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               julia_compat="1.6", preferred_gcc_version=v"5")
+               julia_compat="1.7", preferred_gcc_version=v"5")
```

Comment:
I don't think we should change this, if there's an issue with Pkg in v1.6 that should be addressed somewhere else.